frontend/src/scenes/notebooks/Nodes/NotebookNodeInsight.tsx

```diff
@@ -31,11 +34,25 @@ export const NotebookNodeInsight = createPostHogWidgetNode<NotebookNodeInsightAt
     startExpanded: true,
     attributes: {
         id: {},
+        title: {},
```

Comment:
I'd be in favour of making this a default attribute similar to what we do for `height` and `nodeId`:
https://github.com/PostHog/posthog/blob/cc674bd339e7b15b40dbf22d68aba14a66315e42/frontend/src/scenes/notebooks/Nodes/NodeWrapper.tsx#L224-L225